tests/js/spec/views/dashboardsV2/metricsWidgetQueries.spec.tsx

```diff
@@ -163,4 +162,43 @@ describe('Dashboards > MetricsWidgetQueries', function () {
       })
     );
   });
+
+  it('does not re-fetch when renaming legend alias', () => {
+    const mock = MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/metrics/data/',
+      body: TestStubs.MetricsField({field: SessionMetric.SENTRY_SESSIONS_SESSION}),
+    });
+    const children = jest.fn(() => <div data-test-id="child" />);
```

Comment:
do we need a `data-test-id` here?

Reply:
We don't, it was a leftover from copy-paste, good catch!